common/util.c

```diff
@@ -121,7 +122,8 @@ bool read_private_key(uint8_t *buf, size_t len, yh_algorithm *algo,
     *algo = YH_ALGO_EC_ED25519;
 
     if (internal_repr == true) {
-#if OPENSSL_VERSION_NUMBER < 0x10101000L
+#if (OPENSSL_VERSION_NUMBER < 0x10101000L) ||                                  \
+  (OPENSSL_VERSION_NUMBER == 0x20000000L)
```

Comment:
use LIBRESSL_VERSION_NUMBER here as well?

Reply:
Fixed in 2df470a50b2e9359e055cb1bf837f0f95a1e72c5